ext/soap/tests/bugs/bug38067.phpt

```diff
@@ -32,13 +32,13 @@ class TestSoapClient extends SoapClient {
 
 $client = new TestSoapClient(__DIR__.'/bug38067.wsdl',
     array('encoding' => 'ISO-8859-1'));
-$str = 'test: �';
+$str = 'test: �';
```

Comment:
Unintended change? Did you editor treat this file as UTF-8 instead of ISO-8859-1?

Reply:
Yes, it's surely an editor issue.